source/fuzz/fact_manager.cpp

```diff
@@ -515,6 +534,26 @@ class FactManager::DataSynonymAndIdEquationFacts {
   // in that relation.
   std::unordered_map<const protobufs::DataDescriptor*, OperationSet>
       id_equations_;
+
+  // Represents a map from a strongly connected component (SCC) in the graph to
```

Comment:
"the graph" is abrupt - can you write a few words to explain what we have a graph for?

Reply:
Fixed in f989f1e.